mdbenchmark/tests/data/analyze-files-namd/2/bench.job

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -l
+# Standard output and error:
+#SBATCH -o ./apoa1.out.%j
+#SBATCH -e ./apoa1.err.%j
+# Initial working directory:
+#SBATCH -D ./
+# Job Name:
+#SBATCH -J apoa1
+#
+# Queue (Partition):
+#SBATCH --partition=express
+#
+# Number of nodes and MPI tasks per node:
+#SBATCH --nodes=2
+#SBATCH --ntasks-per-node=32
+# Wall clock limit:
+#SBATCH --time=00:20:00
+
+module purge
+module load impi
+module load namd
+
+# Run namd for 15 minutes
+namd2 apoa1
```

Comment:
This is very minor, but the syntax for the `namd2 apoa1` call is not the one that we put into `templates/draco`. Maybe we should put them to the same state :)